features/payment/managing_payment_methods/editing_payment_method.feature

```diff
@@ -0,0 +1,48 @@
+@managing_payment_methods
+Feature: Editing payment methods
+    In order to change which payment methods are available in my store
+    As an Administrator
+    I want to be able to edit payment method
+
+    Background:
+        Given the store has a payment method "Offline" with a code "offline"
+        And I am logged in as an administrator
+
+    @ui
+    Scenario: Renaming the payment method
+        Given I want to modify a payment method "Offline"
+        When I rename it to "Cash on delivery" in "English (United States)"
+        And I save my changes
+        Then I should be notified about successful edition
+        And this payment method name should be "Cash on delivery"
+
+    @ui
+    Scenario: Changing gateway
+        Given I want to modify a payment method "Offline"
+        When I rename it to "Paypal Express Checkout" in "English (United States)"
+        And I choose "Paypal Express Checkout" gateway
+        And I save my changes
+        Then I should be notified about successful edition
+        And this payment method gateway should be "paypal_express_checkout"
```

Comment:
It should be.. but there is a problem with gateways and relating it with payment method, it will be fixed in future. But for now, I guess that filter for gateway code to convert it to name would be an unnecessary code.